bundle/rhdh/manifests/rhdh-default-config_v1_configmap.yaml

```
@@ -403,7 +409,7 @@ data:
     type: Opaque
     stringData:
       .npmrc: |
-        @redhat:registry=https://npm.registry.redhat.com
+        @redhat:registry=https://npm.stage.registry.redhat.com
```

probably don't want to link to the STAGE repo from our production builds... ? 
```suggestion
        @redhat:registry=https://npm.registry.redhat.com
```

---

Hey @nickboldt, based on the recent discussion with RHDH mgt team, we would wait until the plugins are available in NPM production environment. Once it's ready and validated, we'll update this PR to point to production. 

@elai-shalev , please could you convert to it to draft PR and that should add a _do-not-merge_ label.